src/main/java/by/byshnev/services/ArticleService.java

```diff
@@ -0,0 +1,9 @@
+package by.byshnev.services;
+
+import by.byshnev.models.Article;
+
+public interface ArticleService {
```

Comment:
creating interfaces for decoupling different layers of application - correct approach. usually web app consists of 3 main layers:
* Transport layer - implementation of your API. Common example - controllers. It's responsibility - send and receive data. Should not contain any business logic
* Service layer - here all business logic lives (e.g. use repositories to fetch data, processing it, mapping to transport dtos)
* Data access layer - it's where actual data comes from (e.g. repository)

Create service implementation which uses DAO to get articles, use it in your controller